codespell_lib/data/dictionary.txt

```diff
@@ -34651,45 +34660,59 @@ nealy->nearly, newly,
 neares->nearest
 nearset->nearest
 neast->nearest, beast,
+necasserily->necessarily
 necassery->necessary
+necassrily->necessarily
 necassry->necessary
 necause->because
 neccecarily->necessarily
 neccecary->necessary
 neccesarily->necessarily
 neccesary->necessary
 neccessarily->necessarily
+neccessarrily->necessarily
 neccessarry->necessary
 neccessary->necessary
 neccessities->necessities
 neccessity->necessity
+neccisarily->necessarily
 neccisary->necessary
+neccsessarily->necessarily
 neccsessary->necessary
 necesarily->necessarily
 necesarrily->necessarily
 necesarry->necessary
 necesary->necessary
+necessaerily->necessarily
 necessaery->necessary
+necessaily->necessarily
 necessairly->necessarily
 necessar->necessary
 necessarally->necessarily
 necessaraly->necessarily
+necessariily->necessarily
 necessarilly->necessarily
 necessarilyn->necessarily
 necessariy->necessary, necessarily,
+necessarlily->necessarily
 necessarly->necessary, necessarily,
+necessarrily->necessarily
 necessarry->necessary
 necessaryly->necessarily
 necessay->necessary
 necesserily->necessarily
 necessery->necessary
+necessesarily->necessarily
 necessesary->necessary
 necessiate->necessitate
 nechanism->mechanism
+necssarily->necessarily
 necssary->necessary
 ned->need, end,
 nedd->need
 nedded->needed
+nedding->needing
```

Comment:
Probably `nodding`, perhaps even `wedding` and `bedding`?

Reply:
I'll add `nodding`. I couldn't find the other two typos.